We do not need to check formatting for this project.

--- StrEnum-0.4.15/setup.py.orig
+++ StrEnum-0.4.15/setup.py
@@ -20,7 +20,6 @@
     extras_require={
         "test": [
             "pytest",
-            "pytest-black",
             "pytest-cov",
         ],
         "docs": [
--- StrEnum-0.4.15/pytest.ini.orig
+++ StrEnum-0.4.15/pytest.ini
@@ -1,3 +1,3 @@
 [pytest]
-addopts = --cov=strenum --cov-report term-missing --black --ignore=versioneer.py --ignore=strenum/_version.py --ignore=docs/
+addopts = --cov=strenum --cov-report term-missing --ignore=versioneer.py --ignore=strenum/_version.py --ignore=docs/
 filterwarnings = ignore::DeprecationWarning
